src/tests/ov_helpers/ov_snippets_models/src/subgraph_mha.cpp

```diff
@@ -158,6 +158,65 @@ std::shared_ptr<ov::Model> MHAFunction::initReference() const {
 
     return std::make_shared<ov::Model>(NodeVector{subgraph}, ngraphParams);
 }
+
+std::shared_ptr<ov::Model> MHA2DFunction::initOriginal() const {
+    auto transpose0Param = std::make_shared<ov::opset1::Parameter>(precisions[0], input_shapes[0]);
```

Comment:
Minor: transpose* prefix is not needed here since there are no transpose layers in this builder
the same for reference builder

Reply:
Removed in both functions